composer.json

```diff
@@ -54,7 +54,7 @@
     "oat-sa/extension-tao-lti": "15.13.1",
     "oat-sa/extension-tao-ltideliveryprovider": "12.16.0",
     "oat-sa/extension-tao-revision": "10.7.1",
-    "oat-sa/extension-tao-mediamanager": "12.36.0",
+    "oat-sa/extension-tao-mediamanager": "12.36.0.1",
```

Comment:
- `v12.36.0.1` introduces a dependency to tao-item with `>=v12.1.0 || >=v12.0.2.1` ([mediamanager diff](https://github.com/oat-sa/extension-tao-mediamanager/compare/v12.36.0...v12.36.0.1))
- `v12.37.0` introduces a dependency to tao-item with `>=12.1.0` ([mediamanager diff](https://github.com/oat-sa/extension-tao-mediamanager/compare/v12.36.0...v12.37.0))

So that allows to use tao-item `v12.0.2.1` instead of `v12.1.0`.

tao-item `v12.1.0` brings in some ACL changes and I wanted to avoid introducing more changes than needed, since the associated ticket does not seem to be targeted on the August release:
- https://github.com/oat-sa/extension-tao-item/compare/v12.0.2...v12.1.0
- Changes seem to be related to SOLAR-60 and looks like there is a PR still opened for that
  - As per my understanding, that epic does is not set to be part of the August release, so I was trying to stay on the safe side